Terminal.Gui/App/MainLoop/MainLoopCoordinator.cs

```diff
@@ -56,7 +56,7 @@ IComponentFactory<TInputRecord> componentFactory
     /// <summary>
     ///     Starts the input loop thread in separate task (returning immediately).
     /// </summary>
-    public async Task StartAsync ()
+    public async Task StartInputTask ()
```

Comment:
By convention `async` task methods should have the `Async` suffix. So, I recommend `StartInputTaskAsync` or better `StartInputAsync`.